test/remittance.js

```diff
@@ -0,0 +1,168 @@
+const truffleAssert = require('truffle-assertions');
+const Remittance = artifacts.require('Remittance');
+
+const { toBN, toWei, fromAscii } = web3.utils;
+
+const zeroAddress = '0x0000000000000000000000000000000000000000';
+
+contract('Remittance', accounts => {
+  const [owner, carol, someoneElse] = accounts;
+  let contract;
```

Comment:
I am not sure it is a good idea to shadow the meaning of `contract` found on line 8...

Reply:
OK, let me rename it